packages/gasket-plugin-elastic-apm/lib/index.js

```diff
@@ -46,41 +45,36 @@ module.exports = {
         );
 
         if (!apm.isStarted()) {
-          logger.notice('WARNING Elastic APM agent is not started. Use `--require elastic-apm-node/start`');
+          logger.notice('WARNING Elastic APM agent is not started. Use `--require ./setup.js`');
```

Comment:
`notice` was used for a depecration "notice" before. I think this warrants a `warning` level now
```suggestion
          logger.warning('Elastic APM agent is not started. Use `--require ./setup.js`');
```